cisco_threatgrid/plugin.spec.yaml

```diff
@@ -5,7 +5,7 @@ name: cisco_threatgrid
 title: Cisco ThreatGrid
 description: This plugin retrieves malware reports, report samples and URLs for analysis to Cisco ThreatGrid server
  Cisco ThreatGrid combines advanced sandboxing with threat intelligence into one unified solution to protect organizations from malware
-version: 2.1.0
+version: 3.0.0
```

Comment:
I think this could be a minor/patch version bump given the safety check in your code as well as the required: false within the plugin spec.

Reply:
Changed version to: 2.1.1